README.md

```diff
@@ -770,6 +784,20 @@ They are well suited when you expect that there might be errors in the input tha
 further investigation.
 
 ```c++
+/**
+ * Depending on the underlying system endianness,
+ * Convert possibly broken UTF-8 string into UTF-16LE/UTF-16BE string and stop on error.
+ *
+ * During the conversion also validation of the input string is done.
+ * This function is suitable to work with inputs from untrusted sources.
+ *
+ * @param input         the UTF-8 string to convert
+ * @param length        the length of the string in bytes
+ * @param utf16_buffer  the pointer to buffer that can hold conversion result
+ * @return a result pair struct with an error code and either the position of the error if any or the number of char16_t written if successful.
+ */
+simdutf_warn_unused result convert_utf8_to_utf16_with_errors(const char * input, size_t length, char16_t* utf16_output) noexcept;
```

Comment:
There's a lot more functions that currently use `utf16le` and `utf16be`, not just the UTF8-to-UTF16 conversion functions